test/spicepods/tpch/sf1/federated/databricks[sql_warehouse].yaml

```diff
@@ -1,16 +1,16 @@
 version: v1
 kind: Spicepod
-name: databricks[federated]-sql_warehouse
+name: databricks[sql_warehouse]-federated
```

Comment:
```suggestion
name: databricks[federated]-sql_warehouse
```

@peasee and I set the convention. The "trait" or "version" of it is last.

Reply:
That isn't the convention we're using in the other files, i.e its currently databricks[delta_lake]-federated

Reply:
https://github.com/spiceai/spiceai/blob/trunk/test%2Fspicepods%2FREADME.md

Reply:
The other files are wrong :-)

Reply:
Ah, nevermind, I'm wrong :-)

Reply:
The variant here is the databricks mode, the federated part is to denote it's not accelerated - which matches the written convention